test/unit/org/apache/cassandra/utils/memory/BufferPoolTest.java

```diff
@@ -146,6 +149,32 @@ public void testRecycle()
         requestUpToSize(RandomAccessReader.DEFAULT_BUFFER_SIZE, 3 * BufferPool.NORMAL_CHUNK_SIZE);
     }
 
+    @Test
+    public void testBufferDefaultMaxCapacity()
+    {
+        DatabaseDescriptor.clientInitialization();
+        ByteBuf noMaxCapacity = GlobalBufferPoolAllocator.instance.buffer(100);
```

Comment:
nit: would be nice to release the buffer

Reply:
Done